### PR TITLE
Get server side props fix

### PR DESCRIPTION
**Pull request by Nw84**
We have changed from getStaticProps to getServerSideProps after talking to Richard, so now everything seems to be working as we intended

### CODE REVIEW

**Comment by Nw84** [comment]
Co-authored-by: Mats Hemå [mhema42@users.noreply.github.com](mailto:mhema42@users.noreply.github.com)

Co-authored-by: Simon Ljungström [simonlj8@users.noreply.github.com](mailto:simonlj8@users.noreply.github.com)
Co-authored-by: Emma Anderdal[emmambr@users.noreply.github.com](mailto:emmambr@users.noreply.github.com)

File: pages/movies/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function movies(props) {
     )
 }
 
-export async function getStaticProps() {
+export async function getServerSideProps() {
```
Comment (mhema42) [inline pages/movies/index.js:19]: Perfect that it was possible to change this to GSSP :)

**Review by mhema42 — APPROVED** [review]
Perfect Tobias, this was a really good fix, even if it worked rather ok before, now it´s really nice :)
And I think a cookieHelper is a very good solution, separation of conserns and it´s easy to reuse if another component later on needs to check if we have a logincookie.
Very good job as usual Tobias!

**Review by emmaambr — APPROVED** [review]
💯

**Pull request merged**